windows/win_file.py

```diff
@@ -56,17 +56,26 @@
 
 EXAMPLES = '''
 # create a file
-- win_file: path=C:\\temp\\foo.conf 
+- win_file:
+    path: C:\temp\foo.conf
 
 # touch a file (creates if not present, updates modification time if present)
-- win_file: path=C:\\temp\\foo.conf state=touch 
+- win_file:
+    path: C:\temp\foo.conf
```

Comment:
Double backslashes aren't needed for unquoted YAML, but _are_ needed for `k=v` pairs and when single quoting the path in YAML. This may be a case where we should explicitly put in a `k=v` example showing that.